README.md

```diff
@@ -64,6 +97,19 @@ spec:
 EOS
 ```
 
+
+## Key Features
+
+* Binding with backing services represented by Kubernetes resources including third-party CRD-backed resources.
+* Binding secret generation based on OLM descriptors and CRD/CR annotations.
+* Binding with multiple-backing services.
+* Binding as volume mounts and environment variables.
```

Comment:
afaik for volume mount we have an issue https://github.com/redhat-developer/service-binding-operator/issues/661 . For annotations and descriptors also the `bindAs` is not being used.

Reply:
I believe each feature should have its own bullet:

```md
* Project binding values as volume mounts.
* Project binding values as environment variables.
```

Reply:
@Avni-Sharma This list is more of a "what we intend to support over time" so that a user can make decisions about considering this project for adoption. Therefore, I'm hoping to have this as a static list. What do you think?

As a separate task, we should have detailed docs on the extent of support for each that must vary between releases/PRs.

@isutton +1 will do!